src/pages/workspace/MemberRightIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Badge from '@components/Badge';
+import useLocalize from '@hooks/useLocalize';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+
+type MemberRightIconProps = {
+    owner?: string;
+    role?: string;
+    login?: string;
+};
+
+export default function MemberRightIcon({role, owner, login}: MemberRightIconProps) {
+    const {translate} = useLocalize();
+
+    let badgeText = '';
+    if (owner === login) {
```

Comment:
Should we check role here in addition or instead? I'm wondering if there's a case where both of these are blank